src/index.d.ts

```diff
@@ -0,0 +1,209 @@
+interface DropdownValue {
```

Comment:
we need to define which props are optional and which are required

Reply:
Defined optional parameters